companies/management/commands/importregistrations.py

```diff
@@ -0,0 +1,140 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.core.files.base import ContentFile
+from django.db import transaction
+from django.utils.text import slugify
+from companies.models import Company, Contact
+from people.models import Programme as Program
+from exhibitors.models import Exhibitor, CatalogInfo, \
+        WorkField, JobType, Continent, Value
+from fair.models import Fair
+from urllib.request import urlopen
+from urllib.error import HTTPError
+from ast import literal_eval
+import csv
+
+
+anmalan_url = 'http://anmalan.armada.nu'
+
+
+def get_objects_by_name(Model, names):
+    objects = []
+    for name in names:
+        obj, _ = Model.objects.get_or_create(name=name)
+        objects.append(obj)
+    return objects
+
+
+def eval_list(string):
+    if string:
+        return literal_eval(string)
+    else:
+        return []
+
+
+class Command(BaseCommand):
+    help = 'takes a csv file from anmalan.armada.nu and imports it to database'
+
+    def add_arguments(self, parser):
+        parser.add_argument('file')
+
+    def handle(self, *args, **options):
+        if not options['file']:
+            raise CommandError('Please provide a csv file to import')
+        csv.register_dialect('semi', delimiter=';')
+        with open(options['file']) as file:
+            reader = csv.DictReader(
+                    file, dialect='semi')
+            fair = Fair.objects.get(name='Armada 2016')
+            if fair is None:
```

Comment:
I don't like using the not operator with non-boolean values since it treats "",0,None etc as false. If you have a number for instance it's a huge semantically difference between it being 0 and it being None.